backend/app/api/game_manager.py

```diff
@@ -172,4 +172,19 @@ def asdict(self) -> Dict[str, str]:
             'board':            self.board,
             'gameRunning':      self.game_running,
             'winner':           self.winner,
-        }
\ No newline at end of file
+        }
+
+    def one_random_letter(self, player : Player) -> None:
```

Comment:
This is pretty much just what's written above, I'm not entirely sure that these functions belong here, or at least not under this name. Maybe change it to `extra_letter_perk(self) -> None`.

The GameManager should be aware of the current player through the `current_player` attribute, I'm not sure how the perks work, but I assume you can only activate a perk on your turn. If this is the case, then you should be able to just do `self._players[self.current_player]` to get the current player, you don't need to pass a player into the function. Otherwise, this function looks